doc/MONIT/examples.md

```diff
@@ -0,0 +1,63 @@
+# MONIT go client query examples
+
+## 1. How to use
+1. You can use monit tool directly from `/cvmfs/cms.cern.ch/cmsmon/monit --help`
+2. You can build your own monit go tool.
+    - Firstly download requirements: 
+    
+        `go get github.com/go-stomp/stomp && go get github.com/vkuznet/x509proxy`
```

Comment:
before that user(s) required to setup GOPATH, otherwise it will not work. Please add appropriate notice here.

Reply:
I added a line for how to install golang and how to set go path.